app/api/attachments/routes.js

```diff
@@ -96,6 +97,7 @@ export default app => {
     '/api/attachments/upload',
     needsAuthorization(['admin', 'editor']),
     uploadMiddleware(attachmentsPath),
+    activitylogMiddleware,
     validation.validateRequest(
       Joi.object()
```

Comment:
Maybe take the time to switch from Joi to AJV? In order to get rid of some tech debt with the boy-scout approach.

Reply:
Although to move the validation to AJV is easy, to update the validation tests is not being straightforward, maybe it could be better to change this when we have a definition for #2934